packages/cli/templates/platform-app-tailwind/src/App.tsx

```diff
@@ -0,0 +1,124 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { useEffect, useState } from 'react';
```

Comment:
```suggestion
import { useEffect, useState } from 'react';
```
No `import React` required